gamification/process/update.py

```diff
@@ -135,6 +136,29 @@ def get_participant_objects():
     return participants
 
 
+def update_participants_data_with_meta_review(participants):
```

Comment:
On line 35 (https://github.com/coala/community/pull/190/files#diff-0bee1776ac8d509157f197b459b8b9afR35) you are calling `update_participants_data_with_meta_review(participant)` but here it is called `participants`. So should the argument be one participant or many participants?